src/components/BuddiesContext.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { createContext, useEffect, useState } from "react";
```

Comment:
Since React 18, this is no longer required to be imported:
```suggestion
import { createContext, useEffect, useState } from "react";
```